docs/source/gbd2019_models/risk_effects/sbp/index.rst

```diff
@@ -180,47 +180,48 @@ The risk-outcome pairs listed below are standard GBD relationships. The relative
 
 PAFs and relative risks can be pulled using the following code: 
 
 rrs = get_draws(gbd_id_type='rei_id', gbd_id=107, source='rr', year_id=2019, gbd_round_id=6, status='best', decomp_step='step4') 
 
 pafs = get_draws(gbd_id_type=['rei_id', 'cause_id'], gbd_id=[107, 493], source='burdenator', measure_id=2, metric_id=2, year_id=2019, gbd_round_id=6, status='best', decomp_step='step5') 
 
-Risk Outcome Pair #2
-++++++++++++++++++++
+Risk Outcome Pair #2: Ischemic stroke
++++++++++++++++++++++++++++++++++++++
 
 :ref:`See ischemic stroke documentation <2019_cause_ischemic_stroke>`
 
-The relative risks apply to the incidence rates of acute ischemic stroke. They should be applied using the formula affected_measure*(1 – PAF\ :sub:`r107,c495`\) * RR\ :sub:`r107`\). 
+The relative risks apply to the incidence rates of acute ischemic stroke. They should be applied using the formula incidence(i) = incidence*(1-PAF\ :sub:`r107`\)*RR^{max((SBP_i - TMREL),0)/10}. The relative risk for GBD 2019 is for a 10-unit increase in mm Hg. 
 
 PAFs and relative risks can be pulled using the following code: 
 
 rrs = get_draws(gbd_id_type='rei_id', gbd_id=107, source='rr', year_id=2019, gbd_round_id=6, status='best', decomp_step='step4') 
 
 pafs = get_draws(gbd_id_type=['rei_id', 'cause_id'], gbd_id=[107, 495], source='burdenator', measure_id=2, metric_id=2, year_id=2019, gbd_round_id=6, status='best', decomp_step='step5') 
 
-Risk Outcome Pair #3
-++++++++++++++++++++
+Risk Outcome Pair #3: Intracerebral hemorrhage
+++++++++++++++++++++++++++++++++++++++++++++++
 
 :ref:`See intracerebral hemorrhage documentation <2019_cause_ich>`
 
-The relative risks apply to the incidence rates of acute intracerebral hemorrhage. They should be applied using the formula affected_measure*(1 – PAF\ :sub:`r107,c496`\) * RR\ :sub:`r107`\). 
+The relative risks apply to the incidence rates of acute intracerebral hemorrhage. They should be applied using the formula incidence(i) = incidence*(1-PAF\ :sub:`r107`\)*RR^{max((SBP_i - TMREL),0)/10}. The relative risk for GBD 2019 is for a 10-unit increase in mm Hg. 
+
 
 PAFs and relative risks can be pulled using the following code: 
 
 rrs = get_draws(gbd_id_type='rei_id', gbd_id=107, source='rr', year_id=2019, gbd_round_id=6, status='best', decomp_step='step4') 
 
 pafs = get_draws(gbd_id_type=['rei_id', 'cause_id'], gbd_id=[107, 496], source='burdenator', measure_id=2, metric_id=2, year_id=2019, gbd_round_id=6, status='best', decomp_step='step5') 
 
-Risk Outcome Pair #4
-++++++++++++++++++++
+Risk Outcome Pair #4: Subarachnoid hemorrhage
++++++++++++++++++++++++++++++++++++++++++++++
 
 :ref:`See subarachnoid hemorrhage documentation <2019_cause_sah>`
 
```

Comment:
I believe the following lines related to the RR should have also been updated for subarachnoid hemorrhage like they were for all other causes to say the following:

"The relative risks apply to the incidence rates of acute subarachnoid hemorrhage. They should be applied using the formula incidence(i) = incidence*(1-PAF\ :sub:`r107`\)*RR^{max((SBP_i - TMREL),0)/10}. The relative risk for GBD 2019 is for a 10-unit increase in mm Hg."

Reply:
That follows. Would you want to update this PR (e.g., push a local commit to this branch)?